validator/sawtooth_validator/journal/chain.py

```diff
@@ -505,6 +505,37 @@ def _verify_block(self, blkw):
         self._blocks_processing[blkw.block.header_signature] = validator
         self._executor.submit(validator.run)
 
+    def _remove_block_and_verify_descendants(self, parent_block):
```

Comment:
_remove_block_and_verify_descendants is confusing as a name. '_verify_descendants' is clearer.